frontend/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "jsxBracketSameLine": true
+}
```

Comment:
```suggestion
{
    "semi": true,
    "useTabs": false,
    "endOfLine": "lf",
    "printWidth": 88,
    "trailingComma": "none",
    "jsxBracketSameLine": true,
    "overrides": [
        {
            "files": ["*.js"],
            "options": {
                "tabWidth": 4
            }
        },
        {
            "files": ["*.html", "*.json", "*.prettierrc", "*.css"],
            "options": {
                "tabWidth": 4
            }
        },
        {
            "files": ["*.md"],
            "options": {
                "proseWrap": "always"
            }
        }
    ]
}

```

Reply:
Again, we can't remove semicolons as it would increase amount of potential ambiguities in code.
Line width, tab width, and quotes should be the same as in Python code and ones I recommended in snipped above are the ones used by black formatter.

Reply:
the same situation as in #4 why separate ["*.js"] & ["*.html", "*.json", "*.prettierrc", "*.css"] while "options" are the same?

Reply:
Okay, it may be harmful in cases when markdown files are not ending withj `.md`. Could you please check if it still works when placed in global scope, and if it does I will change my recommendation here and in other places.

Reply:
It indded does (at least I didn't came across any issues while testing), so I belive it is valid option. But i noticed that ESlint is not higlighting anything in `.md` files (i was testing both option, local & global) and had to run `npx prettier filename.md` to test it